### PR TITLE
[W5.4][W14-A3]Shradheya Thakre

### PR DESCRIPTION
**Pull request by tshradheya**

### CODE REVIEW

File: doc/me.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#Shradheya Thakre
```
Comment (500poundbear) [inline doc/me.md]: File not required, please remove from your PR.

File: src/seedu/addressbook/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    @Override
+    public CommandResult execute() {
+        addressBook.sortPersonList();
+        List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+        return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);
+    }
+
+
+
```
Comment (500poundbear) [inline src/seedu/addressbook/commands/SortCommand.java]: Please remove excess empty lines

File: src/seedu/addressbook/parser/Parser.java [hunk 3]
```diff
@@ -90,12 +80,17 @@ public Command parseCommand(String userInput) {
...
+        case SortCommand.COMMAND_WORD:
+            return new SortCommand();
+
         case ViewCommand.COMMAND_WORD:
             return prepareView(arguments);
 
         case ViewAllCommand.COMMAND_WORD:
             return prepareViewAll(arguments);
 
+
```
Comment (500poundbear) [inline src/seedu/addressbook/parser/Parser.java]: Please remove unnecessary blank lines

File: doc/me.md (hunk 1 above)
Comment (tshradheya) [reply doc/me.md]: Incidentally made the [W51.b] LO on the master so it got mixed here too.

Only later I realized that it no longer is a Clean PR

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 4]
```diff
@@ -123,6 +118,18 @@ public void remove(ReadOnlyPerson toRemove) throws PersonNotFoundException {
     }
 
     /**
+     * Sorts list of people by name in lexicographical order
+     */
+    public void sortByName(){
```
Comment (JunQuann) [inline src/seedu/addressbook/data/person/UniquePersonList.java]: Maybe it'll be better to override compareTo method in Name.java to compare Name by string. This comparator can just then use the compareTo method in Name.java

**Comment by alvinlyj** [comment]
Review comments in #260, noticed that PRs #260 and #266 are similar. However PR #260 has your changes on Junit and this PR does not. --> Reason for reviewing PR #260 instead.

Do take note of review comments and close both PRs after reading comments

**Pull request closed without merging**